WordPress/Classes/Extensions/Post+BloggingPrompts.swift

```diff
@@ -0,0 +1,72 @@
+
+extension Post {
+
+    func prepareForPrompt(_ prompt: Prompt?) {
+        guard let prompt = prompt else { return }
```

Comment:
Per the [style guide](https://github.com/wordpress-mobile/swift-style-guide), `return` should be on the next line. i.e.:
```
guard let prompt = prompt else { 
    return 
}
```